packages/cw-orch-core/Cargo.toml

```diff
@@ -18,9 +18,6 @@ all-features = true
 
 [features]
 default = []
-# enable node-backed tests (ensure Docker is running)
-# run with `cargo test --jobs 1 --features node-tests`
-node-tests = []
```

Comment:
can we comment them out instead of removing?

Reply:
This feature does nothing on cw-orch-core, any reason we would want to keep it?

Reply:
ahh oeps didn't see that. thanks!